enarx-assigned

```diff
@@ -1,21 +1,33 @@
 #!/usr/bin/python3
 # SPDX-License-Identifier: Apache-2.0
 
-from github import Github
+import github
 import os
 
-github = Github(os.environ['GITHUB_TOKEN'])
-org = github.get_organization('enarx')
+g = github.Github(os.environ['GITHUB_TOKEN'])
+org = g.get_organization('enarx')
 
 project = {p.name: p for p in org.get_projects(state='open')}['Sprint']
 columns = {c.name: c for c in project.get_columns()}
 
-for issue in github.search_issues(f"org:enarx is:issue state:open is:public -label:conference -project:enarx/{project.number}"):
+for issue in g.search_issues(f"org:enarx is:issue state:open is:public -label:conference -project:enarx/{project.number}"):
     if issue.assignee is not None:
-        columns['Assigned'].create_card(content_id=issue.id, content_type='Issue')
+        try:	
+            columns['Assigned'].create_card(content_id=issue.id, content_type='Issue')
+        except github.GithubException as e:	
+            error = e.data["errors"][0]	
+            if error["resource"] != "ProjectCard" or error["code"] != "unprocessable":	
+                raise	
+            print("Card already in project.")
```

Comment:
I agree with @npmccallum that this could be made into a function or something and re-used in different places.